mozilla-try-scripts/gecko-try.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash -e
+
+######
+# This script allows you to test a new generic-worker Windows release on
+# gecko try. It deploys your chosen generic-worker release to a set of
+# staging worker types that are created for this purpose, in order not
+# to impact other worker types while testing a generic-worker release.
+#
+# TODO: this should be rewritten as e.g. a go program, to reduce number
+# of system dependencies (e.g. bash, curl, jq, grep, sleep, ...).
+#
+# TODO: alternatively, it could probably be written to run in docker, or
+# as a taskcluster task that runs based on some github event.
+#
+######
+
+function open_browser_page {
+  case "$OSTYPE" in
+    linux*)
+      xdg-open "${1}"
+      ;;
+    darwin*)
+      open "${1}"
+      ;;
+  esac
+}
+
+NEW_VERSION="${1}"
+
+if [ -z "${NEW_VERSION}" ]; then
+  echo "Please specify version of generic-worker to use in gecko try push, e.g. '${0}' 10.4.1" >&2
+  exit 64
+fi
+
+echo "Checking system dependencies..."
+for command in curl jq grep sleep file git hg cat sed rm mktemp python go; do
```

Comment:
Awesome.

I think this list isn't particularly onerous either. Most Mozilla/Taskluster people at least will have all of these installed.